distribution/docs/src/main/resources/_contents/_transformers/kml-metacard-transformer-contents.adoc

```diff
@@ -0,0 +1,152 @@
+==== KML Metacard Transformer
+
+The KML Metacard Transformer is responsible for translating a metacard into a KML-formatted document.
+The KML will contain an HTML description that will display in the pop-up bubble in Google Earth.
+The HTML contains links to the full metadata view as well as the product.
+
+===== Installing the KML Metacard Transformer
+
+The `spatial-kml-transformer` feature is installed by default with a standard installation in the ${ddf-spatial} Application.
```

Comment:
No, it's just part of the `spatial-kml` feature. I agree this is confusing. Revised.